test-e2e/project-leave.js

```diff
@@ -345,4 +347,51 @@ test('leaving a project while disconnected', async (t) => {
   )
 })
 
+test('partly-left projects are cleaned up on startup', async (t) => {
```

Comment:
it'd be nice to confirm that data (on disk) is actually cleaned up in this scenario. not sure how complicated that would be to test though

Reply:
Opened #693 to test this. (Doesn't test this _specific_ cleanup case; let me know if you want me to do that.)